2023-05-26 18:46:00,287 - Debug Adapter (main) - INFO - Working directory: /
2023-05-26 18:46:00,287 - Debug Adapter (main) - INFO - Arguments: 
{   'app_flash_off': 65536,
    'board_type': None,
    'cmdfile': None,
    'core_file': (),
    'debug': 3,
    'developer_mode': 'none',
    'device_name': 'esp32',
    'elfpath': (   '/Users/mohamedhossam/Desktop/MohamedHossam/TUM/IoT/iot_praktikum/build/blink.elf',),
    'log_file': '/Users/mohamedhossam/Desktop/MohamedHossam/TUM/IoT/iot_praktikum/debug.log',
    'log_mult_files': None,
    'log_no_debug_console': None,
    'oocd': 'openocd',
    'oocd_args': None,
    'oocd_ip': 'localhost',
    'oocd_mode': 'connect_to_instance',
    'oocd_scripts': None,
    'port': 43474,
    'postmortem': False,
    'tmo_scale_factor': 1,
    'toolchain_prefix': 'xtensa-esp32-elf-'}
2023-05-26 18:46:00,287 - Debug Adapter (main) - CRITICAL - Debug adapter -> Extension: DEBUG_ADAPTER_STARTED
2023-05-26 18:46:00,287 - Debug Adapter (main) - INFO - Starting. Cmd: /Users/mohamedhossam/.vscode/extensions/espressif.esp-idf-extension-1.6.2/esp_debug_adapter/debug_adapter_main.py -d 3 -e /Users/mohamedhossam/Desktop/MohamedHossam/TUM/IoT/iot_praktikum/build/blink.elf -l /Users/mohamedhossam/Desktop/MohamedHossam/TUM/IoT/iot_praktikum/debug.log -p 43474 -dn esp32 -a 0x10000 -t xtensa-esp32-elf-

2023-05-26 18:46:00,293 - Debug Adapter (main) - INFO - Listening on port 43474
2023-05-26 18:46:00,293 - Debug Adapter (main) - CRITICAL - Debug adapter -> Extension: DEBUG_ADAPTER_READY2CONNECT
2023-05-26 18:46:00,954 - Gdb - INFO - NOTIFY: thread-group-added {'id': 'i1'}
2023-05-26 18:46:11,118 - Debug Adapter (main) - ERROR - Failed to connect to "remote 127.0.0.1:3333"!
Traceback (most recent call last):
  File "/Users/mohamedhossam/.vscode/extensions/espressif.esp-idf-extension-1.6.2/esp_debug_adapter/debug_adapter/command_processor.py", line 52, in __call__
    on_request(protocol_message)
  File "/Users/mohamedhossam/.vscode/extensions/espressif.esp-idf-extension-1.6.2/esp_debug_adapter/debug_adapter/command_processor.py", line 75, in on_initialize_request
    raise e
  File "/Users/mohamedhossam/.vscode/extensions/espressif.esp-idf-extension-1.6.2/esp_debug_adapter/debug_adapter/command_processor.py", line 70, in on_initialize_request
    self.da.adapter_init()
  File "/Users/mohamedhossam/.vscode/extensions/espressif.esp-idf-extension-1.6.2/esp_debug_adapter/debug_adapter/debug_adapter.py", line 329, in adapter_init
    raise e
  File "/Users/mohamedhossam/.vscode/extensions/espressif.esp-idf-extension-1.6.2/esp_debug_adapter/debug_adapter/debug_adapter.py", line 325, in adapter_init
    self.start_gdb()
  File "/Users/mohamedhossam/.vscode/extensions/espressif.esp-idf-extension-1.6.2/esp_debug_adapter/debug_adapter/debug_adapter.py", line 659, in start_gdb
    raise e
  File "/Users/mohamedhossam/.vscode/extensions/espressif.esp-idf-extension-1.6.2/esp_debug_adapter/debug_adapter/debug_adapter.py", line 656, in start_gdb
    self._gdb.connect()
  File "/Users/mohamedhossam/.vscode/extensions/espressif.esp-idf-extension-1.6.2/esp_debug_adapter/debug_adapter/debug_backend/debug_backend/gdb.py", line 578, in connect
    self.target_select(remote_mode, self._remote_target, tmo=tmo)
  File "/Users/mohamedhossam/.vscode/extensions/espressif.esp-idf-extension-1.6.2/esp_debug_adapter/debug_adapter/debug_backend/debug_backend/gdb.py", line 258, in target_select
    raise DebuggerError('Failed to connect to "%s %s"!' % (tgt_type, tgt_params))
debug_adapter.debug_backend.debug_backend.defs.DebuggerError: Failed to connect to "remote 127.0.0.1:3333"!
2023-05-26 18:46:11,141 - Debug Adapter (main) - CRITICAL - Debug adapter -> Extension: DEBUG_ADAPTER_STOPPED
